tools/configure.sh

```diff
@@ -134,13 +150,7 @@ if [ ! -d ${configpath} ]; then
     echo "Directory for ${boardconfig} does not exist.  Options are:"
     echo ""
     echo "Select one of the following options for <board-name>:"
-    configlist=`find ${TOPDIR}/boards -name defconfig`
-    for defconfig in ${configlist}; do
-      config=`dirname ${defconfig} | sed -e "s,${TOPDIR}/boards/,,g"`
-      boardname=`echo ${config} | cut -d'/' -f3`
-      configname=`echo ${config} | cut -d'/' -f5`
-      echo "  ${boardname}:${configname}"
-    done
+    dumpcfgs
```

Comment:
Since there's now an explicit list option wouldn't it be better to just indicate "use --list for listing boards"? this way if you mess up the parameters you don't get the long list and just the error instead

Reply:
Should we update configure.c and README.txt?

Reply:
@v01d Done.  Should we keep USAGE after that?

@xiaoxiang781216 both tools/README.txt and the toplevel README.md don't detail the options, I didn't find anything to update there.
Yes configure.c should be updated. Maybe a little bit re-worked.  For instance getopt_long will be used.

Reply:
> @v01d Done. Should we keep USAGE after that?
Sorry, don't understand what you mean.

Reply:
I'm referring to the $USAGE variable, that shows how to use the tool.
It's printed just after what used to be dumpcfgs.

Reply:
I think it is OK to leave as is.